tensionmap.py

```diff
@@ -249,6 +282,7 @@ def draw(self, context):
         row1.active = context.object.data.tm_active
         row1.operator("tm.update_selected")
         row1.prop(context.object.data, "tm_enable_vertex_colors", text="Enable Vertex Colors")
+        row1.prop(context.object.data, "tm_enable_vertex_groups", text="Enable Vertex Groups")
```

Comment:
In Blender, vertex groups appear before vertex color (by default).
So I think the `Enable Vertex Groups` should appear before the `Enable Vertex Colors` in our UI too.

Reply:
Done!